strings/base_activation.h

```diff
@@ -21,6 +21,11 @@ namespace winrt::impl
     template <typename Interface>
     hresult get_runtime_activation_factory(param::hstring const& name, void** result) noexcept
     {
+        if (winrt_activation_handler)
+        {
+            return winrt_activation_handler(*(void**)(&name), guid_of<Interface>(), result);
+        }
+
```

Comment:
This requires the custom activation handler to deal with all possible activation scenarios. In certain kinds of tests that might be an overkill. It would be nice to have a way for the custom handler to return a magic value signifying that it cannot handle that specific activation and the default activation procedure should just continue, something like:

```cpp
        if (winrt_activation_handler)
        {
            if (auto hr = winrt_activation_handler(*(void**)(&name), guid_of<Interface>(), result);
                hr != E_NOTIMPL) {
                    return hr;
                }
        }
        ...

Reply:
I agree - I've had need for this flexibility myself